dependencies.gradle

```diff
@@ -6,4 +6,8 @@ dependencies {
     compileOnly('com.github.GTNewHorizons:EnderIO:2.8.17:dev')
     compileOnly('com.github.GTNewHorizons:Draconic-Evolution:1.3.10-GTNH:dev')
     api('com.github.GTNewHorizons:ForestryMC:4.9.16:dev')
+
+    // For testing in dev environment.
+    devOnlyNonPublishable('com.github.GTNewHorizons:StorageDrawers:1.13.5-GTNH')
+    devOnlyNonPublishable("com.github.GTNewHorizons:waila:1.8.1")
```

Comment:
`devOnlyNonPublishable` == `compileOnly` + `runtimeOnlyNonPublishable`. You don't need drawers and waila to compile this mod, so `runtimeOnlyNonPublishable` is enough. c.f. https://gtnh.miraheze.org/wiki/Development#Dependency_types
I also think that these should be commented out by default to reduce load time in dev env, just like other compat deps listed above.